hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -318,7 +318,9 @@ int MDMParser::waitFinalResp(_CALLBACKPTR cb /* = NULL*/,
                     DEBUG_D("New SMS at index %d\r\n", a);
                     if (sms_cb) SMSreceived(a);
                 }
-                else if ((sscanf(cmd, "CIEV: 9,%d", &a) == 1)) {
+                // TODO: This should include other LTE devices,
+                // perhaps adding _net.act < 7
+                else if ((_dev.dev != DEV_SARA_R410) && (sscanf(cmd, "CIEV: 9,%d", &a) == 1)) {
```

Comment:
My intention here was that if the device is the `SARA_R410`, it will evaluate to `false` and short-circuit the rest of the `&&` expression, not running the `sscanf()`.  As far as I can tell it works because I don't see the `CIEV matched: 9,x` debugging messages with `SARA_R410`.